.github/workflows/release-internal.yml

```diff
@@ -32,20 +32,46 @@ on:
         required: true
         default: false
 
+  schedule:
+    - cron: '0 13 * * 3' # Every Wednesday at 8:00 AM central time
+
 defaults:
   run:
     shell: bash
 
 jobs:
+  get-last-successful-release-run:
+    runs-on: ubuntu-latest
+    outputs:
+      commit_sha: ${{ steps.get_last_successful_release_run.outputs.commit_sha }}
+    steps:
+      - name: Get last successful Release to Cloud run details
+        id: get_last_successful_release_run
+        run: |
+          last_run_url=$(curl --silent -H "Authorization: Bearer ${{ secrets.GITHUB_TOKEN }}" "https://api.github.com/repos/${{ github.repository }}/actions/runs?event=workflow_dispatch&status=success&branch=${{ github.ref }}&workflow_file=release.internal.yml" | jq -r '.workflow_runs[0].url')
+          commit_sha=$(curl --silent -H "Authorization: Bearer ${{ secrets.GITHUB_TOKEN }}" $last_run_url | jq -r '.head_sha')
+          echo "::set-output name=commit_sha::$commit_sha"
+
+  check-for-new-commits:
+    runs-on: ubuntu-latest
+    needs: get-last-successful-release-run
+    outputs:
+      new_commits: ${{ steps.check_new_commits.outputs.new_commits }}
+    steps:
+      - name: Check for new commits since last release
+        id: check_new_commits
+        run: |
+          last_release_date=$(curl --silent "https://api.github.com/repos/${{ github.repository }}/releases/tags/${{ needs.get-last-successful-run.outputs.commit_sha }}" | jq -r .published_at)
```

Comment:
I think we want to retrieve commits not releases (i.e. we may not have a github release yet). Looks like this could work: 
```https://api.github.com/repos/dbt-labs/dbt-bigquery/commits/2c77893d0fb7b5d76ba85cc16712b9563a60329b```

Reply:
updating this more this morning i'm starting to wonder if this should be its own action that triggers our current workflow to supply the fields we need that way. thoughts?